test/remote-test.js

```diff
@@ -1873,3 +1960,499 @@ describe('Remote', function() {
     });
   });
 });
+
+describe.skip('Request API consistency tests', function() {
```

Comment:
Yeah. I'll unskip them when I've generated fixtures. Not sure if they're actually passing ATM. By the way, there's a bug with eslint that prohibits spread operator